### ISSUE TITLE
Update SDK

### ISSUE THREAD

**Issue by pergolafabio** [pergolafabio/Hikvision-Addons#57]
Hey @mion00 

I was trying to update SDK to: EN-HCNetSDKV6.1.9.4_build20220412_linux64  , needed to run on ubuntu now
But got segmentation faults, so gonna wait for now

You had an idea right, about seg fault?

**Comment by mion00** [comment]
Yeah seg fault errors are very hard to debug without access to the source
code.
Generally speaking it is probably C code in the SDK that accesses some
memory pointer outside of its permitted range, but there could be any other
number of reason in addition to that.
Can you attach here a log showing when the seg fault happened?

**Comment by pergolafabio** [comment]
ah, its seems i got another error, bus error

![image](https://user-images.githubusercontent.com/43251136/218567177-d0d07df4-ae7b-4194-9807-15288298e3e5.png)

**Comment by mion00** [comment]
I have some changes coming in another branch that may help with segfaults I think, but I need more time to finish them.
For now you can try this: add the environment variable `PYTHONFAULTHANDLER=true` to activate a module in Python that can print stack traces from C.
